src/modules/launcher/Plugins/Microsoft.Plugin.Uri/Main.cs

```diff
@@ -87,6 +114,13 @@ public List<Result> Query(Query query)
             return results;
         }
 
+        private bool IsActivationKeywordOnlyAndIsDefaultBrowserSet(Query query)
```

Comment:
Divide it into two functions and use them directly in the Query

Reply:
Okay, thank you